example/build_cmake_in_tree/CMakeLists.txt

```
@@ -2,13 +2,18 @@
 # but your project can set it higher
 cmake_minimum_required(VERSION 3.10)
 
-# Declare your project - must have C++ support
+# Project can mix languages - must have C++ support
+# Kokkos flags are only applied to C++ files
 project(Example CXX)
 
+# We build kokkos as a subdirectory of our project
 add_subdirectory(${Example_SOURCE_DIR}/../.. ${Example_BINARY_DIR}/kokkos)
 
-add_executable(example cmake_example.cpp)
+add_executable(example cmake_example.cpp foo.f)
```

```suggestion
add_executable(example cmake_example.cpp)
```
We don't use `Fortran` here anymore.